client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/hooks/use-credentials-form.ts

```diff
@@ -126,14 +130,24 @@ export const useCredentialsForm = (
 
 	const submitWithApplicationPassword = useCallback(
 		async ( siteId: number, from: string, siteInfoResult: UrlData ) => {
-			if ( isWPCOM( siteInfoResult ) || isNotWordPress( siteInfoResult ) ) {
+			if ( isWPCOM( siteInfoResult ) ) {
+				if ( ! siteSlug ) {
+					return;
+				}
+				await sendTicketAsync( {
+					locale,
+					blog_url: siteSlug,
+					from_url: from,
+				} );
+				onSubmit( siteInfoResult );
+			} else if ( isNotWordPress( siteInfoResult ) ) {
```

Comment:
why are we skipping the ticket creation if the site is not on WordPress? They already got to the flow and clicked submit, so probably worth to follow up with a ticket

Reply:
I think we could do that as a follow-up. WDYT?

Reply:
@valterlorran I think we should deal with that as part of this PR since it's semantically related.

>  so probably worth to follow up with a ticket
probably this phrase created confusion, I meant a follow up Zendesk ticket, not a follow up issue/PR

Reply:
Nope, I got it. The problem is that we'll need to work on the back end to add the information that the site is not a WordPress site. That's why I said that it would be worth working on a follow-up PR as this PR will grow significantly and will require a back-end PR to make it work.

Reply:
Oh, ok. Make sure to make a follow up issue detailing the work needed to be done. 🙇